### ISSUE TITLE
Avoid double-slash between host url and path?

### ISSUE THREAD

**Issue by ghost** [Mermade/widdershins#54]
Input:
```json
{
  "swagger" : "2.0",
  "info" : {
    "description" : "This is documentation of the My API",
    "version" : "0.1",
    "title" : "My API"
  },
  "host" : "api.myurl.com",
  "tags" : [ {
    "name" : "Something",
    "description" : "Endpoint for managing something"
  } ],
  "schemes" : [ "https" ],
  "paths" : {
    "/api/somethings" : {
      "get" : {
        "tags" : [ "Something" ],
        "summary" : "getSomething",
        "description" : "Fetch a list of somethings",
        "operationId" : "getSomething",
        "consumes" : [ "application/json" ],
        "produces" : [ "application/json" ],
        "parameters" : [ {
          "name" : "limit",
          "in" : "query",
          "description" : "Limit the number of results",
          "required" : false,
          "type" : "integer",
          "default" : 25,
          "maximum" : 10000.0,
          "minimum" : 1.0,
          "format" : "int32"
        } ],
        "responses" : {
          "200" : {
            "description" : "successful operation"
          }
        }
      }
    }
  },
  "definitions" : {}
}
```
Output curl example:
```
/```shell
# You can also use wget
curl -X GET https://api.myurl.com//api/somethings
/```
```
Is there an easy way for me to avoid the double slash after .com there? (Other than perhaps `sed 's|.com//|.com/|g'`:)

**Comment by MikeRalphson** [comment]
Good spot! I think a helper function would do the trick...

**Comment by stiancognite** [comment]
So.. That's a change in widdershins or is it something I should do? :) (Or both, with a pointer or two.)

**Comment by MikeRalphson** [comment]
I'll make the change in widdershins (remind me, do you use the `npm` module?). It's not as simple as it first looks getting the `://` not to be replaced and having `{}` characters not get %-encoded...

**Comment by stiancognite** [comment]
I used npm to install widdershins, but I use widdershins as a command line tool. Currently using widdershins v2.2.3.

**Comment by MikeRalphson** [comment]
Should be fixed in v2.2.4

**Comment by stiancognite** [comment]
Nice! Thanks :)

**Comment by scottgerring** [comment]
Has this regressed? Running the example above through Widdershins 3.6.0 gives the same behaviour, e.g.:

```
# You can also use wget
curl -X GET https://api.myurl.com//api/somethings
```

**Comment by MikeRalphson** [comment]
@scottgerring just re-tested it and it looks fine here. Are you using the stock templates?

**Comment by MikeRalphson** [comment]
Regression should be fixed in `v3.6.1`. Thanks for reporting it!
